src/migrations/migrations/28.0.2snap1/1_add-maintenance-window-start.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
This warning will be shown as of v28.0.0, right? Rather than trying to forward-guess the snap version we'll release, I recommend just using the version where this is actually needed: `28.0.0snap1`. Yes, that will never exist, but it will make more sense in a year when we wonder why this feature was introduced in v28.0.2 :stuck_out_tongue: .

Reply:
Ah, didn't know that it would work. Thought it just checks "equals" and not "equals or higher". 😄